addons/mail/static/src/utils/test_utils.js

```diff
@@ -67,9 +144,11 @@ function _useChatWindow(callbacks) {
         destroy: prevDestroy,
     } = callbacks;
     return Object.assign({}, callbacks, {
-        mount: prevMount.concat(() => {
+        mount: prevMount.concat(async () => {
             // trigger mounting of chat window manager
-            Component.env.services['chat_window']._onWebClientReady();
+            await afterNextRender(async () => {
+                await Component.env.services['chat_window']._onWebClientReady();
+            });
```

Comment:
It was just an attempt to fix the issue and it didn't even work.

Reply:
Yup, I see... 😕

Reply:
Removed `afterNextRender` at each step. But then kept `await` of `mount` for security, because `afterNextRender` around mount callbacks is wrapping several of them and might stop after only one of them is mounted if the others are taking significantly longer.